drivers/include/periph/uart.h

```diff
@@ -94,6 +94,17 @@ typedef unsigned int uart_t;
 /** @} */
 
 /**
+ * @brief   parity mode enum
```

Comment:
Just `Parity modes`? The availability depends on hardware and driver implementation.